guests/python/README.md

```diff
@@ -14,7 +14,7 @@ just build-release
 ```
 
 ## Python Version
-We currently bundle [Python 3.14.0], [build for WASI](https://docs.python.org/3/library/intro.html#webassembly-platforms).
+We currently bundle [Python 3.14.1], [build for WASI](https://docs.python.org/3/library/intro.html#webassembly-platforms).
```

Comment:
The `Justfile` looks like we're using `3.14.0` now -- which is fine -- but please keep it in sync with the README (i.e. if you don't update the Python version, you need to take back this change here and the one in line 270).